x-pack/plugins/security_solution/cypress/integration/timeline_templates/creation.spec.ts

```diff
@@ -97,4 +100,23 @@ describe('Timeline Templates', () => {
       cy.get(NOTES).should('have.text', timeline.notes);
     });
   });
+
+  it('Create template from timeline', () => {
+    loginAndWaitForPageWithoutDateRange(TIMELINES_URL);
+    waitForTimelinesPanelToBeLoaded();
+
+    createTimeline(timeline);
+
+    cy.get(TIMELINE_COLLAPSED_ITEMS_BTN).first().click();
+    cy.get(TIMELINE_CREATE_TEMPLATE_FROM_TIMELINE_BTN).click({ force: true });
+
+    cy.wait('@timeline').then(({ request }) => {
```

Comment:
Is there any reason why we are asserting the BE responses instead of checking on the UI that the template is there?

Reply:
As this involves opening up a modal on client side, I'm not sure if this would cause flakiness.

Reply:
But ok, I've added an additional check to see if timeline modal shows up.